diagnostic.md

```diff
@@ -6,70 +6,74 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+1) `resources :books, only: [:show]`
+2) `get '/books/:id' => books:#getbookmethod`
+
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+Inside the railes api directory and then config/routes.rb file.
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+Creates tables in a database.
```

Comment:
It executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.